.github/workflows/label_pull_request_onCreate.yml

```diff
@@ -0,0 +1,21 @@
+# Workflow that labels a pr with the correct label
+name: "[Github Management] Pull Request Labeler on create or taken out of draft"
+
+on:
+  pull_request:
+    types: [opened, ready_for_review]
```

Comment:
We'll also need to add logic to prevent the jobs from running on draft PRs. I believe the `opened` type applies to draft PRs being opened as well

Reply:
added